src/Identity/ApiAuthorization.IdentityServer/src/Data/ApiAuthorizationDbContext.cs

```diff
@@ -17,17 +17,64 @@ namespace Microsoft.AspNetCore.ApiAuthorization.IdentityServer
     /// <summary>
     /// Database abstraction for a combined <see cref="DbContext"/> using ASP.NET Identity and Identity Server.
     /// </summary>
-    /// <typeparam name="TUser"></typeparam>
-    public class ApiAuthorizationDbContext<TUser> : IdentityDbContext<TUser>, IPersistedGrantDbContext where TUser : IdentityUser
+    /// <typeparam name="TUser">The type of the user objects.</typeparam>
+    public class ApiAuthorizationDbContext<TUser> : ApiAuthorizationDbContext<TUser, string>
+        where TUser : IdentityUser
     {
-        private readonly IOptions<OperationalStoreOptions> _operationalStoreOptions;
+        /// <summary>
+        /// Initializes a new instance of <see cref="ApiAuthorizationDbContext{TUser}"/>.
+        /// </summary>
+        /// <param name="options">The <see cref="DbContextOptions"/>.</param>
+        /// <param name="operationalStoreOptions">The <see cref="IOptions{OperationalStoreOptions}"/>.</param>
+        public ApiAuthorizationDbContext(
```

Comment:
Note that this won't be usable with DbContext pooling. If `IOptions<OperationalStoreOptions>` needs to be resolved from the request scope for each context instance, then this is unavoidable. Nevertheless, people will complain.

Reply:
This is a big drawback and can't happen.

Reply:
I don't think I'm understanding this as I haven't yet used DbContext pooling.  That being said, I guess I'm not seeing how this differs from the current implementation.  Could you provide some clearer direction on what a possible solution could be?

Reply:
@StevenRasmussen Why is there a need to add  `IOptions<OperationalStoreOptions> operationalStoreOptions` as a dependency here?

Reply:
This comes from IdentityServer and it has an impact on the DbSchema, so I'm not sure what can we do here.